### ISSUE TITLE
@vue/cli presets to auto include this plugin when creating a new project fails during creation

### ISSUE THREAD

**Issue by wattsie** [forsartis/vue-cli-plugin-tailwind#33]
Hi, just wondering if anyone has been able to use this plugin within a @vue/cli preset for automatic project creation?
Is this supported or do I need to always add it manually once the project is created? (would be nice to do it automatically)

ie. whenever I try to use a vue-cli preset to create a new project, it fails whenever I include this plugin.

If you remove the plugin from the preset, create a new preset defined project, then manually add `vue add tailwind` it works as expected.

PS: If I set "replaceConfig" to false, the `tailwind.config.js` file is not generated.

To Reproduce
Add (or create a new) present in the ~/.vuerc and then create a new project
(ps: I trimmed the paths below)
```
❯ vue create -p Vue3_ALL_Plus2 test5                                                                                                                                                                          

Vue CLI v4.5.11
✨  Creating project in _______/test5.
🗃  Initializing git repository...
⚙️  Installing CLI plugins. This might take a while...

yarn install v1.22.10
info No lockfile found.
[1/4] 🔍  Resolving packages...

success Saved lockfile.
✨  Done in 29.57s.
🚀  Invoking generators...
 ERROR  Error: Error: ENOENT: no such file or directory, unlink '_______/electron/test5/tailwind.config.js'
Error: Error: ENOENT: no such file or directory, unlink '_______/electron/test5/tailwind.config.js'
    at module.exports (_______/electron/test5/node_modules/vue-cli-plugin-tailwind/generator/index.js:86:13)
    at Generator.initPlugins (/usr/local/lib/node_modules/@vue/cli/lib/Generator.js:150:13)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async Generator.generate (/usr/local/lib/node_modules/@vue/cli/lib/Generator.js:168:5)
    at async Creator.create (/usr/local/lib/node_modules/@vue/cli/lib/Creator.js:229:5)
    at async create (/usr/local/lib/node_modules/@vue/cli/lib/create.js:72:3)
```

It appears to try to unlink a non-existant file, might need to check if is exists first before trying to unlink?
ie. in file `vue-cli-plugin-tailwind/generator/index.js` maybe try replacing:
```
fs.unlinkSync(configPath);
```
with
```
if(fs.existsSync(configPath)) {
    fs.unlinkSync(configPath);
}
```

~/.vuerc
```
{
  "useTaobaoRegistry": false,
  "packageManager": "yarn",
  "latestVersion": "4.5.11",
  "presets": {
    "Vue3_ALL_Plus2": {
      "useConfigFiles": true,
      "plugins": {
        "@vue/cli-plugin-babel": {},
        "@vue/cli-plugin-typescript": {
          "classComponent": true,
          "useTsWithBabel": true
        },
        "@vue/cli-plugin-router": {
          "historyMode": false
        },
        "@vue/cli-plugin-vuex": {},
        "@vue/cli-plugin-eslint": {
          "config": "base",
          "lintOn": []
        },
        "@vue/cli-plugin-unit-mocha": {},
        "@vue/cli-plugin-e2e-cypress": {},
        "vue-cli-plugin-moment": {
          "locales": "en"
        },
        "vue-cli-plugin-fontawesome": {},
        "vue-cli-plugin-tailwind": {
          "initConfig": "minimal",
          "replaceConfig": true
        }
      },
      "vueVersion": "3",
      "cssPreprocessor": "dart-sass"
    }
  }
}
```

Many thanks in advance.

**Comment by forsartis** [comment]
Hi, it should work if you remove the `replaceConfig` option from your preset.
```
"vue-cli-plugin-tailwind": {
  "initConfig": "minimal"
}
```
There were a couple of other issues when running this plugin outside the project root. These should be fixed in v2.0.6.

**Comment by wattsie** [comment]
Hi Jens,
  Thanks for the reply. 

  Can confirm latest v2.0.6 does work when installing from a preset.

  Many thanks.

Cheers.
